src/app/backend/resource/common/status.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+// Status provides basic information about resources status on the list.
+type Status struct {
+	// Number of resources that are currently in running state.
+	Running int32 `json:"running"`
+
+	// Number of resources that are currently in pending state.
+	Pending int32 `json:"pending"`
+
+	// Number of resources that are in failed state.
+	Failed int32 `json:"failed"`
+
+	// Number of resources that are in succeeded state.
+	Succeeded int32 `json:"succeeded"`
```

Comment:
I would prefer `map[string]int32` here, because it may be misleading in case of Cron Jobs for example, which are Active or Suspended only. And these labels are not translated anyways.

Reply:
There might be issue with coloring graph slices... Perhaps some constant values for common names like active, failed, running, pending etc. would make it better.

Reply:
I was considering map but it requires introducing yet another portions of strings that would need to be hardcoded in backend and in frontend. Using generic names like that and mapping them to specific resources seems like better idea. Map would complicate the code even more.

Reply:
Syncing frontend with the backend is required at the moment too. You need to keep in mind, that for Cron Job active = running and suspended = failed during development. Both have their own drawbacks. We can stick to the current implementation for now and make the release.